src/components/Datatable/Pagination/NavButton.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconButton } from '../../IconButton';
+import { IconButtonProps } from '../../IconButton/IconButton.types';
+
+const NavButton = styled(IconButton)
+  .attrs((props) => ({
+    ...props,
+  }))
+  .withConfig<IconButtonProps>({
+    shouldForwardProp: (prop) => prop !== 'disabled',
+  })`
+  display: inline-block;
+
+  width: ${pxToRem(48)};
+  height: ${pxToRem(32)};
+  padding: ${pxToRem(8, 16)};
+  margin: ${pxToRem(0, 4)};
+  border-radius: ${pxToRem(3)};
+
+  color: ${({ isDisabled }) =>
+    getColor(isDisabled ? 'graphiteB' : 'graphite2B')};
+  background-color: ${({ isDisabled }) =>
+    getColor(isDisabled ? 'transparent' : 'graphite5H')} !important;
```

Comment:
`!import` is needed? trick with `&` doesn't work?

Reply:
removing it, it just wasn't working before